x/amm/types/pow.go

```diff
@@ -0,0 +1,56 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Don't EVER change after initializing
+// TODO: Analyze choice here.
+var powPrecision, _ = sdk.NewDecFromStr("0.00000001")
```

Comment:
Do we need to have custom pow function? (osmomath that we used to have as separate package does not solve the problem?)

Reply:
We are not utilizing osmomath or osmoutils since they rely on an outdated version of the cosmos SDK, which includes significant changes that would require a substantial amount of time to resolve. Therefore, instead of fixing these two libraries to be compatible with the latest SDK version we use, I opted to selectively extract and incorporate only the necessary functions.

Reply:
Ok